### DISCUSSION TITLE
How to measure voltages greater than 3.3V?

### DISCUSSION THREAD

**Discussion by SpinningPulsar** [fhdm-dev/scoppy#116] [Q&A]
How can you use scoppy to measure voltages greater than 3.3V, can I use a voltage divider to cut down the voltage or will it introduce noise in the circuit?

**Comment by fhdm-dev** [comment]
Hi,

Yes, you can use a voltage divider. I don't think noise will be an issue unless you use very large resistor values.

There are quite a few YouTube videos describing how to use Scoppy with larger voltage ranges, including negative voltages. There are links on [Scoppy home page](https://github.com/fhdm-dev/scoppy) on GitHub.

There is also more information about [Input voltage ranges](https://oscilloscope.fhdm.xyz/wiki/Analog-Front-End.html) and some [example analog front ends](https://oscilloscope.fhdm.xyz/wiki/Analog-Front-End-Examples) in the Scoppy documentation.

You might also want to look at the [Scoppy hub](https://www.hackster.io/scoppy) on Hackster.io
